backend/app/models/product.rb

```diff
@@ -0,0 +1,5 @@
+# frozen_string_literal: true
+
+class Product < ApplicationRecord
+  validates :name, presence: true, length: { maximum: 100 }
```

Comment:
Don't you think we need additional validations? Product could not be created without price, ingredients and weight, could it?

Reply:
What about test product position or temporary position it one of case. What if ingridients will be secret? Or may be not wish public it? About price - what if product in pause to cook
@MikhailZaitsau